zoraxy/umbrel-app.yml

```diff
@@ -0,0 +1,52 @@
+manifestVersion: 1
+id: zoraxy
+name: Zoraxy Proxy Server
+tagline: An efficient reverse proxy server with automated SSL management
+category: networking
+version: "3.1.2"
+port: 8400
+description: >-
+  Zoraxy is a powerful, user-friendly reverse proxy server and forwarding tool, designed to provide a straightforward, flexible, and secure method for managing web traffic within home labs and developer environments. Written in Go, Zoraxy caters to both novice and experienced users by simplifying the process of routing HTTP requests to backend servers, enabling efficient and reliable service management without relying on traditional, more complex servers like NGINX or Apache.

```

Comment:
I'm thinking we should add port instructions and a warning at the top here, what do you think? You can just steal it directly from the nginx proxy manager app:

https://github.com/getumbrel/umbrel-apps/blob/9432ddfefd4914409f36c7f092b749cc2c1dcacc/nginx-proxy-manager/umbrel-app.yml#L8-L17